atd-vze/src/Components/TableSearchBar.js

```diff
@@ -32,27 +34,32 @@ const TableSearchBar = ({ setSearchFilter, clearFilters, fieldsToSearch }) => {
     };
     const queryStringArray = searchQuery();
     searchValue !== "" &&
-      searchFieldValue !== "" &&
+      fieldToSearch !== "" &&
       setSearchFilter(queryStringArray);
   }, [searchValue, setSearchFilter, fieldToSearch, searchFieldValue]);
 
   const handleSearchSubmission = e => {
-    e.preventDefault();
-    setSearchValue(searchFieldValue);
+    if (isFieldSelected) {
+      e.preventDefault();
```

Comment:
Something weird happens when I use "Return" to submit instead of clicking search. The whole page refreshes. Is it possible that we need to move line 43 up before the `if` block?

Reply:
Good call! Even just hitting enter in the field refreshes the page. I moved 43 outside the block, and all is better now.